### ISSUE TITLE
Allow user to set default color for a new note

### ISSUE THREAD

**Issue by gitsnead** [maoschanz/notes-extension-gnome#48]
I have been unable to change the default color for a new note. Can it be done currently in a conf file?

**Comment by maoschanz** [comment]
If you click the "+" button of a red note, the new one will be red, if you click the "+" button of a blue note, the new one will be blue, etc.

The only "default" color is the color of the very first note, which is hardcoded as black in older versions and dark red in the latest version

**Comment by gitsnead** [comment]
Thanks for the explanation. I was unaware that the "+" button would clone properties off the note clicked to create it. That feature makes intuitive sense to me.

I have found an edge case for creating a default color user setting.
A user regularly deletes all her notes upon completing her work for the day. Upon starting work for the day, she creates a new note from the dock icon. The first note of the day will always have an ID of 0 and use the hardcoded value instead of her preferred color.

I have found and changed the hardcoded value to suit my needs. However, if you are interested, I think I could contribute a new feature to handle the use case.
